backends/p4tools/common/lib/trace_event_types.cpp

```diff
@@ -58,7 +58,7 @@ const Expression *Expression::evaluate(const Model &model, bool doComplete) cons
 
 void Expression::print(std::ostream &os) const {
     Generic::print(os);
-    os << ": " << formatHexExpr(value, true);
+    os << ": " << formatHexExpr(value, {true});
```

Comment:
`{true}`: This isn't great for code readability, use designated initializers instead?
```
{.useSep = true}
```

It is technically a C++20 feature, so may not be supported?

Ditto a couple of more times, throughout.

Reply:
Yes, the compiler still uses C++17. :/  But I do not think there is any reason why we can't move to C++20. Would unlock some nice things.

Reply:
My main concern is if this requires bumping the minimal required version of GCC. If so, that could be problematic for downstream tools, or even for users of p4c directly. Not much of C++20 is supported in GCC 9. Designated initializers are, because they were GNU extension for a long time... But this should be probably discussed elsewhere.